ra/proto/ra.proto

```diff
@@ -79,6 +79,12 @@ message AdministrativelyRevokeCertificateRequest {
   // certificate in question. In this case, the keyCompromise reason cannot be
   // specified, because the key cannot be blocked.
   bool malformed = 6;
+  // The CRL shard to store the revocation in. This must match
+  // the CRL shard in the certificate's crlDistributionPoint,
+  // if present.
+  // This is only valid when `malformed` is true; otherwise the shard is
+  // determined by parsing the certificate.
```

Comment:
The two halves of this comment somewhat seem to contradict each other: if it is only valid when `malformed` is true, then we can't parse the cert to determine its crlDistributionPoint, so how can it be required to match?

Reply:
I wrote an update to the language; let me know what you think.